### PR TITLE
Refactor/styleguide+routes

### PR DESCRIPTION
**Pull request by lukem-kainos**
added style guide and removed superfluous routes and controllers

### CODE REVIEW

File: map-demo/app/controllers/Application.java [hunk 1]
```diff
@@ -5,22 +5,11 @@
...
-    public static Result template() {
-
-        return ok(govuk_template.render(new Html("content")));
-    }
-
-    public static Result map() {
-
-        return ok(map.render("Map URL"));
-    }
+    return ok(map.render("Chicken."));
```
Comment (ashleighho-kainos) [inline map-demo/app/controllers/Application.java]: Where is this being rendered?

File: map-demo/app/controllers/Application.java [hunk 2]
```diff
@@ -5,22 +5,11 @@
 
 import play.twirl.api.Html;
 import views.html.*;
```
Comment (sdonaghey03) [inline map-demo/app/controllers/Application.java]: Unused imports

File: map-demo/app/controllers/Application.java (hunk 1 above)
Comment (sdonaghey03) [reply map-demo/app/controllers/Application.java]: Indentation and extra whitespace

**Comment by timja** [comment]
sbt test:checkstyle
[info] Loading project definition from /Users/timja/projects/demapp/map-demo/project
[info] Set current project to map-demo (in build file:/Users/timja/projects/demapp/map-demo/)
java.io.FileNotFoundException: checkstyle-config.xml (No such file or directory)

**Comment by timja** [comment]
Google checkstyle can be found at:
https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml

**Comment by timja** [comment]
Fixed it although there's some checkstyle errors

File: map-demo/app/controllers/Application.java [hunk 3]
```diff
@@ -5,22 +5,11 @@
...
-import views.html.layouts.govuk_template;
 
 public class Application extends Controller {
 
-    public static Result index() {
+  public static Result index() {
 
-        return ok(map.render("Chicken."));
```
Comment (ghost) [inline map-demo/app/controllers/Application.java:14]: 😂

**Comment by timja** [comment]
$ sbt compile
[info] Loading project definition from /Users/timja/projects/demapp/map-demo/project
[info] Set current project to map-demo (in build file:/Users/timja/projects/demapp/map-demo/)
[info] Compiling 2 Scala sources and 1 Java source to /Users/timja/projects/demapp/map-demo/target/scala-2.11/classes...
[error] Checkstyle warning found in /Users/timja/projects/demapp/map-demo/app/controllers/Application.java:3: Using the '._' form of import should be avoided - play._.
[error] Checkstyle warning found in /Users/timja/projects/demapp/map-demo/app/controllers/Application.java:4: Using the '._' form of import should be avoided - play.mvc._.
[error] Checkstyle warning found in /Users/timja/projects/demapp/map-demo/app/controllers/Application.java:7: Using the '._' form of import should be avoided - views.html._.
[error] 3 issue(s) found in Checkstyle report: /Users/timja/projects/demapp/map-demo/target/checkstyle-report.xml

**Comment by timja** [comment]
👍 Thanks for doing this

**Comment by timja** [comment]
👍

**Comment by sdonaghey03** [comment]
👍

**Pull request merged**